.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and verify PR Title
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This would run on the main branch right ? but we would like to run it for pull request so that the pull request title is verified before merging